<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenRGBDMXSettingsEntryUi</class>
 <widget class="QWidget" name="OpenRGBDMXSettingsEntryUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>263</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>E131 settings entry</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="BrightnessLabel">
        <property name="text">
         <string>Brightness Channel:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="NameEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="UniverseComboBox"/>
      </item>
      <item row="5" column="0" rowspan="2" colspan="4">
       <widget class="QListWidget" name="LedWidget"/>
      </item>
      <item row="7" column="0" colspan="4">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QPushButton" name="AddLedButton">
           <property name="text">
            <string>Add LED</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="RemoveButton">
           <property name="text">
            <string>Remove</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="AddEffectButton">
           <property name="text">
            <string>Add Effect</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="ProtocolLabel">
        <property name="text">
         <string>Protocol:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="NameLabel">
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="ProtocolComboBox"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="KeepaliveTimeEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="PortLabel">
        <property name="text">
         <string>Port:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="KeepaliveTimeLabel">
        <property name="text">
         <string>Keepalive Time:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="BrightnessSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="UniverseLabel">
        <property name="text">
         <string>Universe :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="PortEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
